packages/asc-core/src/utils/README.md

```diff
@@ -0,0 +1,49 @@
+Styles utility functions
+
+### getThemeColor
+...
+
+### fillSVG
+...
+
+### fromProps
+Gets value from the props object. Useful for styled-components
+
+```js static
+import styled from 'styled-components'
+import { fromProps } from './utils/fromTheme'
+
+const Div = styled.div`
+  display: ${fromProps('display')};
+`
+
+<Div display="flex" /> // styles: { display: 'flex' }
+```
+
+### fromTheme
+Same as `fromProps`, but it will get a value from the (global) theme object. Useful for styled-components
+
+```js static
+import styled from 'styled-components'
+import fromTheme from './utils'
+
+const theme = {
+  colour: {
+    brand: '#f00'
+  },
+  fontSizes: {
+    title: 2
+  },
+  spacing: {
+    max: 4
+  }
+}
+
+const Div = styled.div`
+  background-color: ${fromTheme('colour.white')};
+  font-size: ${fromTheme('fontSizes.title')}em;
```

Comment:
I like this solution! however I see one potential problem here: typescript cannot automatically check this,  plus editors / IDE's cannot autofill. I'm not sure how big of a problem this would be though, assuming you've already used this in a previous project?

Reply:
True, no IDE support. But it definitely outweighs the burden of having to use deeply nested objects, like so:
```jsx
background-color: ${({ theme: { colour: { buttons: { primary } }  }}) => primary};
```

Reply:
Definitely, but then to support IDE's, perhaps we could make helper functions like getTypography, getColor etc? I'm trying to figure out how we can get best of both worlds...

Reply:
@chrisvanmook I'll merge this now and create an issue so that the type validation can be addressed later.